src/status_im/subs/wallet/wallet.cljs

```diff
@@ -443,27 +451,31 @@
    (let [tx-type       (:tx-type send-data)
-         tokens        (->> (:tokens account)
-                            (map
-                             (fn [token]
-                               (assoc token
-                                      :bridge-disabled?  (and (= tx-type :tx/bridge)
-                                                              (send-utils/bridge-disabled? (:symbol
-                                                                                            token)))
-                                      :networks          (cond->> (network-utils/network-list token
-                                                                                              networks)
-                                                           chain-ids
-                                                           (filter #(some #{(:chain-id %)} chain-ids)))
-                                      :available-balance (utils/calculate-total-token-balance token)
-                                      :total-balance     (utils/calculate-total-token-balance
-                                                          token
-                                                          chain-ids))))
-                            (filter (fn [{:keys [networks]}]
-                                      (pos? (count networks))))
-                            (remove #(when hide-token-fn (hide-token-fn constants/swap-tokens-my %))))
+         tokens        (->>
+                         (:tokens account)
+                         (map
+                          (fn [token]
+                            (assoc token
+                                   :bridge-disabled?  (and (= tx-type :tx/bridge)
+                                                           (send-utils/bridge-disabled? (:symbol
+                                                                                         token)))
+                                   :networks          (cond->> (network-utils/network-list
+                                                                (get token-supported-chains
+                                                                     (:symbol token))
+                                                                networks)
+                                                        chain-ids
+                                                        (filter #(some #{(:chain-id %)} chain-ids)))
+                                   :available-balance (utils/calculate-total-token-balance token)
+                                   :total-balance     (utils/calculate-total-token-balance
+                                                       token
+                                                       chain-ids))))
+                         (filter (fn [{:keys [networks]}]
+                                   (pos? (count networks))))
+                         (remove #(when hide-token-fn (hide-token-fn constants/swap-tokens-my %))))
```

Comment:
We can save another iteration by joining this `filter` + `remove` into a single one. (actually we can even join the previous `map` by using `keep`, but it's highly probable that the readability will be worse).